testing/e2e/src/routes/api.audio.ts

```
@@ -0,0 +1,44 @@
+import { createFileRoute } from '@tanstack/react-router'
+import { generateAudio } from '@tanstack/ai'
+import { createAudioAdapter } from '@/lib/media-providers'
+import type { Feature, Provider } from '@/lib/types'
```

ESLint

[error] `@/lib/types` type import should occur before import of `@/lib/media-providers`

(import/order)